README.md

```diff
@@ -42,11 +42,21 @@ $ cmake ../
 $ make
 ```
 
+If you have built openscap locally it is possible to use it through custom CMake definitions:
+
+```console
+$ cmake -DOPENSCAP_LIBRARIES:PATH=/local/openscap.so/filepath/ \
+    -DOPENSCAP_INCLUDE_DIRS:PATH=/local/openscap/include/path \
+    -DOPENSCAP_VERSION:STRING="X.Y.Z" \
+    ../
+$ make
+```
+
```

Comment:
The new additions and the instructions with `libopenscap.pc` a few lines above, seem to work the same.
I suggest to add it as another option to run SCAP Workbench with locally compiled openscap.

And actually, the existing instructions should note that the system should be able to find the library.
Either by running `ldconfig` or configuring `LD_LIBRARY_PATH`.

Reply:
New commit explains better on how to set environment variables for the linker. Also splits some sections to explicitly have sections for building against locally built OpenSCAP and rpm installed OpenSCAP libraries.